src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -185,6 +185,60 @@ export const exchangeStitchingEnvironment = ({
     },
   }
 
+  const paymentDeviceResolver = {
+    fragment: gql`
+      fragment CommerceOrderPaymentMethod on CommerceOrder {
+        creditCardId
+        bankAccountId
+        paymentMethod #TODO: one of these fields should probably have a different name
+      }
+    `,
+
+    resolve: async (parent, _args, context, info) => {
+      const { creditCardId, bankAccountId, paymentMethod } = parent
```

Comment:
It seems like the Exchange `paymentMethod` hasn't been exposed and used. I wonder if it's possible to extend the `CommerceOrder` interface and just completely replace the Exchange `paymentMethod` string with our new type.

Reply:
That was my thought, but i wondered if we would want to keep a string representation so we don't have to query a bank account or credit card from gravity.

Reply:
For now it is a new field - `paymentDevice`. @damassi, do you know if it is possible to simply replace the exchange string enum field?

Reply:
Not sure if it's a better name, but how about `paymentMethodDetails`?